packages/syft/src/syft/core/tensor/autodp/row_entity_phi.py

```diff
@@ -44,15 +45,16 @@ class RowEntityPhiTensor(PassthroughTensor, ADPTensor):
     # a list of attributes needed for serialization using RecursiveSerde
     __attr_allowlist__ = ["child"]
 
-    def __init__(self, rows: Any, check_shape: bool = True):
+    def __init__(self, rows: Sequence, check_shape: bool = True):
         """Initialize a RowEntityPhiTensor
 
         rows: the actual data organized as an iterable (can be any type of iterable)
         check_shape: whether or not we are already confident that the objects in iterable
             'rows' all have the same dimension (check if we're not sure).
 
         """
-
+        # Container type heirachy: https://docs.python.org/3/library/collections.abc.html
+        self.child: Sequence
```

Comment:
We can actually tell mypy that the instance variable .child will be a Sequence so it can chill out. The other ways are:
```python
# this looks ugly and exposes the parent implementation
def init...
    self.child = rows

# technically works but also allows a child on the Class which we dont want
class RowEntityPhiTensor:
    child: Sequence

    def init...
```

Reply:
Oh that's super helpful, I didn't know about the Sequence data type but reading about it now, I think it should work!